<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UrlAddressBar</class>
 <widget class="QWidget" name="UrlAddressBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>26</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>26</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>26</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="toolButton_Left">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="minibrowserres.qrc">
       <normaloff>:/zuo.png</normaloff>:/zuo.png</iconset>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_Right">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="minibrowserres.qrc">
       <normaloff>:/you.png</normaloff>:/you.png</iconset>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_Refresh">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="minibrowserres.qrc">
       <normaloff>:/shuaxin.png</normaloff>:/shuaxin.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_TimerRefresh">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="minibrowserres.qrc">
       <normaloff>:/shishi.png</normaloff>:/shishi.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
     <property name="arrowType">
      <enum>Qt::NoArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_Url"/>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_QRCode">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="icon">
      <iconset resource="minibrowserres.qrc">
       <normaloff>:/erweima.png</normaloff>:/erweima.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>17</width>
       <height>17</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_Collection">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="icon">
      <iconset>
       <activeoff>:/guanzhu.png</activeoff>
       <activeon>:/xiazai12.png</activeon>
      </iconset>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="minibrowserres.qrc"/>
 </resources>
 <connections/>
</ui>
